packages/eas-cli/src/run/android/aapt.ts

```diff
@@ -0,0 +1,53 @@
+import spawnAsync, { SpawnResult } from '@expo/spawn-async';
+import glob from 'fast-glob';
+
+import Log from '../../log';
+import { getAndroidSdkRootAsync } from './sdk';
+
+async function aaptAsync(...options: string[]): Promise<SpawnResult> {
+  try {
+    return await spawnAsync(await getAaptExecutableAsync(), options);
+  } catch (error: any) {
+    if (error.stderr) {
+      Log.error(error.stderr);
+    }
+    throw error;
+  }
+}
+
+export async function getAaptExecutableAsync(): Promise<string> {
+  const sdkRoot = await getAndroidSdkRootAsync();
+  if (!sdkRoot) {
+    Log.debug('Failed to resolve the Android SDK path, falling back to global aapt executable');
+    return 'aapt';
+  }
+  const aaptPaths = await glob(`${sdkRoot}/build-tools/**/aapt`);
```

Comment:
Use `path.join` instead of interpolation.

Reply:
It's better than interpolation because you don't have to think whether `sdkRoot` ends with `/` or not.